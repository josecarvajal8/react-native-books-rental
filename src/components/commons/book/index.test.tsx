import React from 'react';
import renderer from 'react-test-renderer';
import Book from '.';
const mockProps = {
    author: 'John doe',
    title: 'The GitHub Journey',
    img: 'http://wolox-training.s3.amazonaws.com/uploads/6963511-M.jpg'
}
test('Render Book component correctly', () => {
    const BookCmp = renderer.create(<Book {...mockProps} />).toJSON();
    expect(BookCmp).toMatchSnapshot();
});