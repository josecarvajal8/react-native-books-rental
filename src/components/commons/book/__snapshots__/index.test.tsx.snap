// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Book component correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "uri": "http://wolox-training.s3.amazonaws.com/uploads/6963511-M.jpg",
      }
    }
    style={
      Object {
        "height": 119.10714285714286,
        "marginLeft": 54.34782608695652,
        "width": 90.57971014492753,
      }
    }
  />
  <View
    style={
      Object {
        "marginLeft": 36.231884057971016,
        "marginTop": 7.444196428571429,
      }
    }
  >
    <Text>
      The GitHub Journey
    </Text>
    <Text>
      John doe
    </Text>
  </View>
</View>
`;

exports[`Render booksList component correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "uri": "http://wolox-training.s3.amazonaws.com/uploads/6963511-M.jpg",
      }
    }
    style={
      Object {
        "height": 119.10714285714286,
        "marginLeft": 54.34782608695652,
        "width": 90.57971014492753,
      }
    }
  />
  <View
    style={
      Object {
        "marginLeft": 36.231884057971016,
        "marginTop": 7.444196428571429,
      }
    }
  >
    <Text>
      The GitHub Journey
    </Text>
    <Text>
      John doe
    </Text>
  </View>
</View>
`;
